src/ReservationItem.php

```diff
@@ -512,7 +512,9 @@ public static function showListSimple()
         ]);
 
         foreach ($iterator as $data) {
-            $values[$data['itemtype']] = $data['itemtype']::getTypeName();
+            if (class_exists($data['itemtype']) && $data['itemtype']::canView()) {
```

Comment:
Will be safer to check that it is a `CommonDBTM`.
```suggestion
            if (is_a($data['itemtype'], CommonDBTM::class, true) && $data['itemtype']::canView()) {
```